test/Libraries/DynamoPythonTests/PythonTestsWithLogging.cs

```diff
@@ -18,6 +19,7 @@ public class PythonTestsWithLogging : DynamoModelTestBase
         protected override void GetLibrariesToPreload(List<string> libraries)
         {
             libraries.Add("DSCPython.dll");
+            libraries.Add("DSCoreNodes.dll");
```

Comment:
why this change?

Reply:
needed because of the new class I added in DSCoreNodes. The new class needs to be loaded so that Evaluation can work